trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super.slice(0, 10)
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=base%20Decorator%20class.-,Implement%20a%20method%20correct_name%20that%20makes%20sure%20that%20the%20output%20of%20%40nameable.correct_name%20has%20a%20maximum%20of%2010%20characters.%20If%20it%27s%20longer%20it%20should%20trim%20the%20word.,-See%20your%20decorators), Please makes sure that the output of @nameable.correct_name has a maximum of 10 but If it's longer it should trim the word. so I think you should add a condition here if the word has a length more than 10 so trim if shorter just return the word ✅🔥 so kindly add that condition 💪🏻✨